pkg/zos_api_light/zos_api.go

```diff
@@ -32,12 +29,7 @@ type ZosAPI struct {
 	inMemCache             *cache.Cache
 }
 
-func NewZosAPI(manager substrate.Manager, client zbus.Client, msgBrokerCon string) (ZosAPI, error) {
-	sub, err := manager.Substrate()
-	if err != nil {
-		return ZosAPI{}, err
-	}
-	defer sub.Close()
+func NewZosAPI(client zbus.Client, farmerID uint32, msgBrokerCon string) (ZosAPI, error) {
```

Comment:
where is it used?

Reply:
this will affect zos4 and zoslight code plz handle it there? also why did u get it from the caller instead of the environment?

Reply:
yes, this will affect zos4 and zoslight, will handle it after it's merged and released, also yes we can set the farmer id in the environment and use it instead of the caller passing it.